client/landing/gutenboarding/components/domain-picker/button.tsx

```diff
@@ -65,28 +67,27 @@ const DomainPickerButton: FunctionComponent = () => {
 	const [ freeDomainSuggestions, paidDomainSuggestions ] = partition( suggestions, 'is_free' );
 
 	return (
-		<Button
-			className="domain-picker__button"
-			onClick={ () => setDomainPopoverVisibility( s => ! s ) }
-		>
-			<div className="domain-picker__site-title">
-				{ siteTitle ? siteTitle : NO__( 'Create your site' ) }
-			</div>
-			<div>{ head( freeDomainSuggestions )?.domain_name }</div>
+		<>
+			<Button
+				className={ classnames( 'domain-picker__button', { 'is-open': isDomainPopoverVisible } ) }
+				aria-pressed={ isDomainPopoverVisible }
+				onClick={ () => setDomainPopoverVisibility( s => ! s ) }
+			>
+				{ /* This empty span gives us a good place to anchor our popover */ }
+				<span ref={ popoverAnchor } />
+				{ head( freeDomainSuggestions )?.domain_name ?? '\u00a0' }
```

Comment:
This disappears while we fetch new suggestions. I think we'll need to store in component and update when we get new suggestions.

Don't worry about it for now, we'll need to handle user domain selection as well.

Reply:
Where do Gutenberg or Calypso anchor popovers? (Without checking, I'm assuming they're just appending to `<body>`).

Reply:
"anchor" is for positioning the popover in the window.
I belive the `Slot` (see #37736) determines where the DOM is actually inserted, and I'm not sure at the moment.